### PR TITLE
Tooltips: Consistency and contrast: Issue #11180

### PR DESCRIPTION
**Pull request by timwright12**
## Description
Tooltips: Consistency and contrast issues
Fixes https://github.com/WordPress/gutenberg/issues/11180

## How has this been tested?
Tested in the browser, precommit hooks and e2e tests.

## Screenshots
screen shot 2018-11-16 at 3 44 31 pm
screen shot 2018-11-16 at 3 44 58 pm
screen shot 2018-11-16 at 3 45 03 pm

## Types of changes
Introduces a prop to tooltips passed through iconButtons to control the label positioning.

## Checklist:
- [ x ] My code is tested.
- [ x ] My code follows the WordPress code style.
- [ x ] My code follows the accessibility standards.
- [ x ] My code has proper inline documentation.

### CODE REVIEW

**Review by tofumatt — CHANGES_REQUESTED** [review]
This looks great, actually! Could you run the snapshot test update (using `npm run test-unit:update`) to get the tests passing, and also remove the `gutenberg-mobile` git submodule?

After that I think this would be good to merge, though it'd be great if you could add some screenshots as well. 😄

**Comment by timwright12** [comment]
@tofumatt totally forgot this one existed, nice pick up! I removed the `gutenberg-mobile` submodule, updated the snapshot and added some screenshots to the ticket.

Let me know if there's anything else! Thanks again!

**Review by tofumatt — APPROVED** [review]
Awesome, thanks! 👍

**Pull request merged**
